4-Shaders-Uniforms/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
```

Comment:
Dont think i've seen comments in an assemblyinfo file before. Did you add them? If these are standard comments just ignore this comment.

Reply:
This is just the default comments that show up in that file